metadata-models/src/main/pegasus/com/linkedin/identity/CorpUserStatus.pdl

```diff
@@ -20,5 +20,11 @@ record CorpUserStatus {
   /**
    * Audit stamp containing who last modified the status and when.
    */
+  @Searchable = {
+    "/time": {
+        "fieldName": "statusLastModifiedAt",
+        "fieldType": "COUNT"
+      }
+  }
```

Comment:
you need a backfill job otherwise it won't work for existing deployments. This would mean for a month the charts will be empty.

Reply:
im not writing a backfill job - please feel free to take over.